src/SFML.Graphics/VertexArray.cs

```diff
@@ -111,13 +112,14 @@ public void Clear()
         ////////////////////////////////////////////////////////////
         /// <summary>
         /// Resize the vertex array
-        /// 
-        /// If \a vertexCount is greater than the current size, the previous
+        /// </summary>
+        /// <remarks>
+        /// If vertexCount is greater than the current size, the previous
         /// vertices are kept and new (default-constructed) vertices are
         /// added.
-        /// If \a vertexCount is less than the current size, existing vertices
+        /// If vertexCount is less than the current size, existing vertices
```

Comment:
```suggestion
        /// If <see cref="VertexCount"/> is less than the current size, existing vertices
```

Reply:
I don't know that this works/has a purpose.  The <param> tag provides information about the vertexCount argument.